app/components/donation/credit-card.js

```diff
@@ -81,7 +79,6 @@ export default Component.extend({
 
   actions: {
     submit() {
-      this.set('isSubmitting', true);
```

Comment:
This was actually useless property we don't need anymore, so I would keep this one change in no matter what.